README.md

```diff
@@ -25,6 +25,23 @@ with the documentation generation system.
 
 - [tk-doc-generator style guide](https://developer.shotgunsoftware.com/tk-doc-generator/19bd79d8/?title=Introduction)
 
+## Adding content
+
+For a new file to show up on the site, you'll need to add it to the Table of Contents in the left sidebar:
+
+1. Add it to [`docs/_data/toc.yml`](https://github.com/shotgunsoftware/developer.shotgunsoftware.com/blob/master/docs/_data/toc.yml). Use a meaningful token to identify your page, using dashes to separate words, for example, `pipeline-integrations`.
+2. In [`docs/_data/en/toc_text.yml`](https://github.com/shotgunsoftware/developer.shotgunsoftware.com/blob/master/docs/_data/en/toc_text.yml), provide the English translation for the token you used, as you want it to show up in the left sidebar, for example, "Pipeline Integrations".
+3. Create the file in the proper place in the directory structure. Add metadata to the top, using the identifier you used in `toc.yml`, and following this model:
+
+```
+---
+layout: default
+title: Pipeline Integrations
+pagename: pipeline-integrations
+lang: en
+---
+```
+
```

Comment:
My only suggestion is that we could expand on this a little with an info box on special characters.  This has been a sticking point a couple times, and while we have a ticket in to improve the error message / CI status around this, clearer documentation could be beneficial as well.  Maybe something like:

```suggestion

{% include info title="Special Characters in Titles" content="If using special characters (including colons) in page titles, it's necessary to wrap the title in ASCII double quotes." %}
```

Reply:
will take your content, but change it to markdown so it actually looks right with github.com. thanks!